_data/people/johannbrehmer.yml

```
@@ -6,16 +6,33 @@ institution: New York University
 website: https://johannbrehmer.github.io
 photo: /assets/images/team/Johann-Brehmer.jpg
 presentations:
+  - title: Normalizing flows and the likelihood ratio trick in particle physics
+    date: 2020-01-06
+    url:
+    meeting: Deep learning seminar
+    meetingurl:
+    location: University of Bremen, Germany
+    focus-area: as
+    project: madminer
+  - title: Mining gold: Improving simulation-based inference with latent information (poster)
```

```suggestion
  - title: "Mining gold: Improving simulation-based inference with latent information (poster)"
```

Colons not allowed in YAML unquoted strings.

---

@johannbrehmer, you have disabled the "allow maintainers to push to your PR" option so I can't fix this for you. Please fix it.

---

Thanks @henryiii!

---

:+1: